content/venue/index.md

```diff
@@ -25,6 +25,57 @@ Nørre Allé 7<br>
 
-## Getting there
+## Getting to CPH
 
-TBA
+### I'm arriving by train
+
+You're likely to arrive at either Central Station (aka: "Hovedbanegården" or "København H") or Nørreport Station. Once there you can refer to the "Getting around CPH" section below.
+
+### I'm flying into CPH
+
+Copenhagen Airport aka "Kastrup Airport" is located 12 minutes away by train to the Central Station in Copenhagen's city centre. You need a ticket for 3 zones which can be purchased from one of the automated vending machines or the ticket counter inside the airport atrium. This costs DKK 36 for a single journey.
+
+The Copenhagen Metro also connects Kastrup with central Copenhagen, with trains leaving every 4 minutes during the day and every 15 minutes at night, taking 14 minutes to the city centre. The ticket price is the same of DKK36.
+
+The airport has different stations for mainline trains (Københavns Lufthavn, Kastrup Station) and for the metro (Lufthavnen Station), both are in Terminal 3. Another metro station is named Kastrup but has nothing to do with the airport although it is relatively close.
+
+## Getting around CPH
+
+The two big transport hubs are Central Station (aka: "Hovedbanegården" or "København H") with S-trains, intercity trains, the metro and buses, and Nørreport Station with S-trains, metro, regional trains and buses. Travel by train, bus and metro can be scheduled electronically through [rejseplanen.dk](https://rejseplanen.dk/).
+
+### The CPH Zone system and tickets
+
+The whole city, as well as the surrounding region is divided into fare zones. The range of a single zone can be roughly translated to around seven minutes in the Metro or fifteen minutes in a bus, but always check the zone maps in the stations, some stations are closer to zone borders than others
+
+The number of available ticket types may be bewildering but if you're coming for the conference you should opt for the "single-ride" ticket.
+
+The price of a single-ride ticket depends on the number of zones your travel through, the cheapest is the two-zone ticket which costs DKK 24 for adults (DKK 12 for children under the age of sixteen). This allows you to travel around Copenhagen in two zones; the zone where you stamped or purchased the ticket plus one adjacent zone.
+
+You can travel for one hour starting from the time you stamp it. Within this hour you can switch freely between all trains, Metro, and buses. As long as your last trip starts before the time is up; your ticket will be timestamped in fifteen minute intervals.
+
+### Getting to the venue
```

Comment:
```suggestion
### How to plan your trip to the venue using public transport
```

I noticed that you are writing a how-to here :D

(maybe I'm work damaged by Diataxis haha)